### ISSUE TITLE
Communicate to screen reader users when a highlight is made

### ISSUE THREAD

**Issue by klemay** [hypothesis/client#2060]
[this is a usability improvement and not part of WCAG compliance work]

When screen reader users highlight text, no notification is given to the user. When text is marked for annotation a change is communicated by shifting focus into the annotation text box. However, with highlights, no information is given. 

Since the user can review the text and hear what text is annotated, this finding was not included in the VPAT. But some indication, for example through the use of a message within a live region, could be beneficial for notifying users that they have successfully highlighted a section of the text.

**Comment by lyzadanger** [comment]
This should be covered by https://github.com/hypothesis/client/pull/4533
